src/encord_active/cli/utils/coco.py

```diff
@@ -69,6 +70,7 @@ def import_coco_project(images_dir: Path, annotations_file: Path, target: Path,
     coco_importer.create_project(
         dataset=dataset,
         ontology=ontology,
+        ssh_key_path=app_config.get_or_query_ssh_key(),
```

Comment:
Now that I think of this, we shouldn't really force the user to add and ssh key if they only intend to use the product locally.

Can we replace this add the key only if set in the config without prompting the user if it isn't set?

Reply:
Replaced with just a getter!